testData/diagnostics/need_space_after_int.nut line = (2) column = (9) : Error: malformed number
local f = 333
println([5f][1])
         ^

